### PR TITLE
Refactor a bit VersionService.java

### PR DESCRIPTION
**Pull request by jvoisin**
- Tighten the exceptions
- Remove a useless global variable
- Use try-with-resource

### CODE REVIEW

File: airsonic-main/src/main/java/org/airsonic/player/service/VersionService.java [hunk 1]
```diff
@@ -249,7 +244,7 @@ private void readLatestVersion() throws IOException {
             return;
         }
 
-        List<String> unsortedTags = JsonPath.read(content, JSON_PATH);
+        List<String> unsortedTags = JsonPath.read(content, "$..tag_name");
```
Comment (eharris) [inline airsonic-main/src/main/java/org/airsonic/player/service/VersionService.java]: I think it was clearer with `JSON_PATH`.

**Comment by jvoisin** [comment]
Then there isn't anything to change here :)

**Comment by eharris** [comment]
The "try-with-resource" was still a good change! :-)

**Comment by jvoisin** [comment]
It will eventually show up again, it's ok :)

**Pull request closed without merging**